card/mod/standard/set/type/cardtype.rb

```diff
@@ -1,7 +1,7 @@
 include_set Abstract::WqlSearch
 
 def wql_hash
-  { type_id: id }
+  { type_id: id, sort: :name }
```

Comment:
A long time ago we got rid of sorting because it was too expensive. Maybe here?

Reply:
Unfortunately, card queries still sort by default:
```
        def order_config
          @mods[:sort].blank? ? "update" : @mods[:sort]
        end
```

We discussed getting rid of sorting, but I guess we never did it. One cost of losing default sorting is that our default paging behavior might get weird.  If things aren't sorting by anything in particular, how do you know the items on page 1 won't show up again on page 2?

I'm still a little iffy on the rationale for making cardtypes have their own searches rather than using associated search cards like `<Cardtype>+*type+by_name`, but I know one of the most common cases now is JSON exports, and in all the cases I was looking at for those, name sorting seemed a more useful default than date sorting.  Specifically, this made the large Company.json and Metric.json dumps easier to stay oriented in.